.github/actions/validate-issue/index.js

```diff
@@ -0,0 +1,121 @@
+const fs = require("fs");
+const path = require("path");
+const { getInput, setOutput, setFailed } = require("@actions/core");
+const { context, getOctokit } = require("@actions/github");
+const execSync = require("child_process").execSync;
+const semver = require("semver");
+
+(async function() {
```

Comment:
remember top-level await is allowed on node16+, so you could just define a function and then then call it.  😄